splitio/api/commons.py

```diff
@@ -113,4 +115,41 @@ def build_fetch(change_number, fetch_options, metadata):
         extra_headers[_CACHE_CONTROL] = _CACHE_CONTROL_NO_CACHE
     if fetch_options.change_number is not None:
         query['till'] = fetch_options.change_number
-    return query, extra_headers
\ No newline at end of file
+    return query, extra_headers
+
+class APIException(Exception):
+    """Exception to raise when an API call fails."""
+
+    def __init__(self, custom_message, status_code=None):
+        """Constructor."""
+        Exception.__init__(self, custom_message)
+        self._status_code = status_code if status_code else -1
+
+    @property
+    def status_code(self):
+        """Return HTTP status code."""
+        return self._status_code
+
+class HttpClientException(Exception):
```

Comment:
this exception is only thrown by the HTTP clients, i think it belongs to that module

Reply:
will move